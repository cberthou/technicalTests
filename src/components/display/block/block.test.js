import React from 'react';
import { shallow } from 'enzyme';
import Block from './block';

describe('src/components/display/block', () => {
    it('should display the account balance', () => {
        const wrapper = shallow(<Block>Content</Block>);

        expect(wrapper).toMatchSnapshot();
    });
});
