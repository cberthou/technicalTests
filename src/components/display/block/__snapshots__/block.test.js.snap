// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`src/components/display/block should display the account balance 1`] = `
<div>
  Content
</div>
`;
